_includes/profile.html

```diff
@@ -1,8 +1,16 @@
-<div class="profile">
-    <div class="profile-picture">
+<div class="container-fluid profile">
+    <div class="row justify-content-center">
         {% for item in site.data.fellows %}
-            <img src="./assets/img/{{ item.img }}">
+        <div class="col-sm-6 col-lg-2 col-md-3 text-center">
+            <img class="profile-img" src="./assets/img/{{ item.img }}" alt="{{item.name}}">
```

Comment:
Rather than putting an `img` tag inside a `div` I'd recommend using the `background-image` CSS rule to place the images while still preserving the aspect ratio. This might also fix the issues you were having with the carousel.